services/validatorWorker/follower.js

```diff
@@ -16,22 +16,39 @@ async function tick(adapter, iface, channel) {
 
 	// there are no unapproved NewState messages, only merge all eventAggrs
 	const { accounting, newAccounting } = await producer.tick(iface, channel)
+	const ourLatestAccounting = newAccounting || accounting
+
+	let isChannelExhausted = false
```

Comment:
Should this be false or
```
sumMap(toBNMap(ourLatestAccounting)).eq(new BN(channel.depositAmount))
```
If a leader keeps sending invalid `NewState` after a channel is exhausted the follower will keep sending `Heartbeats`

But if we change to the line above we also have to consider `HEALTH_THRESHOLD_PROMILLES `
because the follower accounting can differ by a little bit